### ISSUE TITLE
Audit and standardize how time is tracked in the Moped Database

### ISSUE THREAD

**Issue by frankhereford** [cityofaustin/atd-data-tech#9240]
## TLDR

The moped database does not use a unified pattern with respect to storing time values. This is bonafide tech debt, and a refactor will be required at some point. 

## Details

In the course of reviewing @mddilley's work for Moped [PR 653](https://github.com/cityofaustin/atd-moped/pull/653) and looking at the way that he is wrangling time zones and date storage, it got me wondering how we are doing more broadly at this task.  I dumped the current schema and grepped out all the definitions of timestamp columns to see our _status quo_, (`pg_dump --schema-only | grep timestamp` and the results are interesting. Please see the list below.

We have a about 1/3 of our timestamp fields designated to be in UTC, blind to timezones, and the remaining in timezone-aware timestamp fields. When we use a default value, we have about half using `now()` which is database transaction aware, and about half are `clock_timestamp()` which is transaction unaware. While we don't **explicitly** use transactions at the present time, `graphql-engine` uses them under-the-hood to allow for multiple SQL queries to be bundled up into one graphql exchange. 

In my opinion, this will operate under the usual tech-debt rules. This is one of those things that isn't a problem until it is, and the challenge of untangling the problem is proportional to how long we let it go. 

## See Also

https://xkcd.com/1883/

## Goal

Ideally, all of these would be of the same type, and for those with a default value set, the function used to provide that value should be the same function.

```
    date_added timestamp with time zone DEFAULT clock_timestamp(),
    date_added timestamp with time zone DEFAULT clock_timestamp() NOT NULL,
    created_at timestamp without time zone DEFAULT now()
    created_at timestamp without time zone DEFAULT now(),
    next_retry_at timestamp without time zone,
    created_at timestamp with time zone DEFAULT now() NOT NULL,
    response_received_at timestamp with time zone,
    created_at timestamp without time zone DEFAULT now()
    scheduled_time timestamp with time zone NOT NULL,
    created_at timestamp without time zone DEFAULT now(),
    next_retry_at timestamp with time zone,
    created_at timestamp without time zone DEFAULT now()
    scheduled_time timestamp with time zone NOT NULL,
    created_at timestamp without time zone DEFAULT now(),
    next_retry_at timestamp with time zone,
    occurred_at timestamp with time zone DEFAULT now() NOT NULL,
    upgraded_on timestamp with time zone NOT NULL,
    created_at timestamp with time zone DEFAULT now(),
    date_added timestamp without time zone DEFAULT now() NOT NULL,
    date_added timestamp with time zone DEFAULT clock_timestamp(),
    last_updated timestamp with time zone DEFAULT clock_timestamp()
    date_added timestamp with time zone DEFAULT clock_timestamp(),
    date_added timestamp with time zone DEFAULT clock_timestamp()
    date_added timestamp with time zone DEFAULT clock_timestamp(),
    date_added timestamp with time zone DEFAULT clock_timestamp(),
    date_added timestamp with time zone DEFAULT clock_timestamp(),
    date_created timestamp with time zone DEFAULT clock_timestamp() NOT NULL,
    date_added timestamp with time zone DEFAULT clock_timestamp(),
    date_added timestamp with time zone DEFAULT clock_timestamp(),
    updated_at timestamp with time zone DEFAULT now(),
    create_date timestamp with time zone DEFAULT now() NOT NULL,
    date_added timestamp with time zone DEFAULT clock_timestamp(),
    date_added timestamp with time zone DEFAULT clock_timestamp(),
    date_added timestamp with time zone DEFAULT clock_timestamp()
    created_at timestamp without time zone DEFAULT now()
    date_added timestamp with time zone DEFAULT clock_timestamp(),
```

**Comment by mddilley** [comment]
Awesome, @frankhereford! Thanks for making this issue - it perfectly captures some of the schema quirks around dates and times.
